packages/css/src/components/accordion/accordion.template.ts

```diff
@@ -0,0 +1,60 @@
+import { Accordion } from '@dso-toolkit/sources';
+
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+
+import { accordionHandleHeading } from './accordion-handle-heading.template';
+
+export function accordionTemplate({ variant, sections, reverseAlign }: Accordion) {
+  return html`
+    <div class="dso-accordion ${classMap({
+      [`${variant}`]: !!variant,
+      'dso-align-viewer': !!reverseAlign
+    })}">
+      ${sections.map(section => html`
+        <div class="dso-accordion-section ${classMap({
+          [`dso-${section.state}`]: !!section.state,
+          'dso-open': !!section.open,
+          'dso-nested-accordion': !!section.sections
+        })}">
+          ${accordionHandleHeading(section.header, section, reverseAlign)}
+
+          ${section.open && html`
```

Comment:
Hoe maak jij de afweging tussen `[voorwaarde] && toon_template` en 'n ternary `[voorwaarde] ? toon_template : nothing` ?

Reply:
Ik zie in dit geval geen verschil in de uiteindelijke markup, was het niet zo dat de `&& toon_template` een lege <tag> achter kon laten? Ik heb hier gekozen voor de kortste notatie, maar als ik door het project zoek zie ik dat we de voorkeur geven aan de `? toon_template : nothing` variant.

Kun je nog even kort onderbouwen waarom we daar de voorkeur aan geven?

Reply:
Omdat lege waardes nog steeds waardes zijn die invloed hebben op markup: https://lit.dev/docs/v1/lit-html/writing-templates/#rendering-nothing

Reply:
Duidelijk, ik ga ze omzetten naar de `? toon_template : nothing` constructie